docs/source/models/pic.rst

```diff
@@ -64,20 +64,12 @@ Electro-Magnetic PIC Method
 See :ref:`section Finite-Difference Time-Domain Method <model-AOFDTD>` describing how Maxwell's equations are discretized on a mesh in PIConGPU.
 
 The distribution of **Particles** is described by the distribution function :math:`f_s(\mathbf{x},\mathbf{v},t)`.
-This distribution function is sampled by *markers* (commonly referred to as *macro-particles*).
+This distribution function is sampled by *markers*, which are commonly referred to as *macroparticles*.
+These markers represent blobs of incompressible phase fluid moving in phase space.
 The temporal evolution of the distribution function is simulated by advancing the markers over time according to the Vlasov--Maxwell--Equation in Lagrangian frame (see eq. :eq:`VlasovMaxwell` and [EulerLagrangeFrameOfReference]_).
-
-Markers carry a spatial shape of order :math:`n` and a delta-distribution in momentum space.
-In most cases, these shapes are implemented as B-splines and are pre-integrated to *assignment functions* :math:`S` of the form:
-
-.. math::
-
-   S^0(x) = \big\{ \substack{1 \qquad \text{if}~0 \le x \lt 1\\ 0 \qquad \text{else}}
-
-   S^n(x) = \left(S^{n-1} * S^0\right)(x) = \int_{x-1}^x S^{n-1}(\xi) d\xi
-
-PIConGPU implements these up to order :math:`n=4`.
-The three dimensional marker shape is a multiplicative union of B-splines :math:`S^n(x,y,z) = S^n(x) S^n(y) S^n(z)`.
+A marker has a finite-size and a velocity, such that it can be regarded as a cloud of particles, whose center of mass is the marker's position and whose mean velocity is the marker's velocity.
+The cloud shape :math:`S^n(x)` of order :math:`n` of a marker describes its charge density distribution.
+See :ref:`section Hierarchy of charge assignment schemes <model-shapes>` for a list of available marker shapes in PIConGPU.
```

Comment:
typo: `Hierarchy` -> `hierarchy`

Reply:
I will correct to the correct title with capital nouns.